pkg/apis/eventing/v1alpha1/trigger_types.go

```diff
@@ -54,9 +54,13 @@ type Trigger struct {
 
 var (
 	// Check that Trigger can be validated, can be defaulted, and has immutable fields.
-	_ apis.Validatable   = (*Trigger)(nil)
-	_ apis.Defaultable   = (*Trigger)(nil)
-	_ apis.Immutable     = (*Trigger)(nil)
```

Comment:
Grouped these `var` statements, similar to be consistent with the pattern in majority of other files.